File: src/domain/enums/MuscleGroup.ts
/**
 * High-level muscle groups (browsing/filtering taxonomy).
 * The finer muscle-head taxonomy is modelled per-muscle in M1
 * (see DATA_MODEL.md). This enum is the stable, UI-facing grouping.
 */
export enum MuscleGroup {
  Chest = 'CHEST',
  Back = 'BACK',
  Shoulders = 'SHOULDERS',
  Biceps = 'BICEPS',
  Triceps = 'TRICEPS',
  Forearms = 'FOREARMS',
  Core = 'CORE',
  Quadriceps = 'QUADRICEPS',
  Hamstrings = 'HAMSTRINGS',
  Glutes = 'GLUTES',
  Calves = 'CALVES',
  Neck = 'NECK',
}

File: backend/src/main/java/com/musclemap/catalog/MuscleGroup.java
package com.musclemap.catalog;

/**
 * High-level muscle groups (browsing / filtering taxonomy). Mirrors the frontend
 * {@code MuscleGroup} enum value-for-value, so {@code name()} is the wire format
 * the API and the bundled static dataset share.
 */
public enum MuscleGroup {
    CHEST,
    BACK,
    SHOULDERS,
    BICEPS,
    TRICEPS,
    FOREARMS,
    CORE,
    QUADRICEPS,
    HAMSTRINGS,
    GLUTES,
    CALVES,
    NECK,
    ADDUCTORS,
    ABDUCTORS
}
